packages/manager/src/views/WebPropertyDetailPage/WebPropertyDetailPage.tsx

```diff
@@ -38,34 +46,27 @@ import {
   Thead,
   Tr
 } from '@patternfly/react-table';
-import {
-  CogIcon,
-  CubeIcon,
-  GithubIcon,
-  PackageIcon,
-  PlusCircleIcon,
-  TimesCircleIcon
-} from '@patternfly/react-icons';
-import { useDebounce, useFormatDate, useTabs, useToggle, usePopUp } from '@app/hooks';
-import { pageLinks } from '@app/links';
 
 import toast from 'react-hot-toast';
-import { Ephemeral } from './components/Ephemeral';
-import { EmptyInfo } from './components/EmptyInfo';
 import { Dashboard } from './components/Dashboard';
+import { EmptyInfo } from './components/EmptyInfo';
+import { Ephemeral } from './components/Ephemeral';
 import { AddDeplyoment } from './components/addDeployment';
 
+import { Settings } from '../Settings/Settings';
+
 const URL_LENGTH_LIMIT = 100;
 const INTERNAL_ACCESS_URL_LENGTH = 25;
 
 export const WebPropertyDetailPage = (): JSX.Element => {
   const { query } = useRouter();
+  const initialTab = query.initialTab as string;
   const [isRowExpanded, setIsRowExpanded] = useState<Record<string, boolean>>({});
   const [searchTerm, setSearchTerm] = useState('');
   const [filterByEnv, setFilterByEnv] = useState('');
   const propertyIdentifier = (query?.propertyIdentifier as string) || '';
   const formatDate = useFormatDate();
-  const { openTab, handleTabChange } = useTabs(4);
+  const { openTab, handleTabChange } = useTabs(4, parseInt(initialTab || '0', 10));
```

Comment:
```suggestion
  const { openTab, handleTabChange } = useTabs(4, Number(initialTab || '0', 10));
```

// parsing:
parseInt("20px");       // 20
parseInt("10100", 2);   // 20
parseInt("2e1");        // 2

// type conversion
Number("20px");       // NaN
Number("2e1");        // 20, exponential notation

Reply:
Fixed